content/python/concepts/dictionaries/terms/clear/clear.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: '.clear()'
+Description: 'Removes all entries in a dictionary.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Dictionaries'
+  - 'Methods'
```

Comment:
```suggestion
  - 'Dictionaries'
  - 'Functions'
  - 'Methods'
```

What do we think about adding "Functions" here in the `Tags`? Is it possible some readers might search for these as a Python _function_ as opposed to a _method_?